tests/worker/memory.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::atomic::{AtomicU16, Ordering};
+use wasm_bindgen_test::{wasm_bindgen_test, wasm_bindgen_test_configure};
+
+static COUNTER: AtomicU16 = AtomicU16::new(12);
+
+#[wasm_bindgen_test]
+pub fn test_atomic_var_after_changed() {
+    assert_eq!(COUNTER.load(Ordering::SeqCst), 15);
+}
```

Comment:
How do we guarantee that this is always executed after the other test?

Reply:
Yes I also thought it. The unit tests should be isolated from other. But I had to test the atomics in workers. Because someone can implement the same functionalities using separate workers for each test. Then the atomics will not work like in `nodejs` and `browser`. However I removed it due to bad use of unit tests.